src/reminders/remind_router.erl

```diff
@@ -24,6 +25,12 @@ send_test(User, Channel, HolidayDate) ->
   {ok, Pid} = supervisor:start_child(remind_router_sup, []),
   gen_server:cast(Pid, {send_reminder, User, Channel, HolidayDate, Message, true}).
 
+send_message(User, Channel, Message) ->
```

Comment:
if you introduce this generic send_message function then send_test should not exist anymore and use the new one to test the reminder

Reply:
its not possible because `send_test` accepts `HolidayDate`, and `send_message` sets the data internally. Maybe we need something more general like `notification_handler`, with a function `send(User, Channel, MessageType, Message)`. Where `MessageType` can in fact be a reminder, a test, a notification, o something else in the future.

Reply:
Yeah but what I mean is you should have one `send_message` function that takes all the parameters. Then you can have a higher level function like `send_reminder` that builds the reminder specficic message and then calls`send_message`. `send_test` could be removed, I think it's enough to just call the generic `send_message` with the test specific message.